File: MiSegundaAplicacion/app/src/main/java/es/iesnervion/avazquez/misegundaaplicacion/MainActivity.java
package es.iesnervion.avazquez.misegundaaplicacion;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import java.util.Date;

public class MainActivity extends AppCompatActivity
implements View.OnClickListener
{

    Button btn ;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        btn = findViewById(R.id.botonHorario);
        btn.setOnClickListener(this);

    }

    @Override
    public void onClick(View v) {
        actualizar();

    }

    public void actualizar(){
        btn.setText(new Date().toString());
    }
}

File: PrimerAplicacionKotlin/app/src/main/java/es/iesnervion/avazquez/primeraplicacionkotlin/MainActivity.kt
package es.iesnervion.avazquez.primeraplicacionkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity(), View.OnClickListener{

    var txtResultado: TextView ? = null;

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val botonEnviar = findViewById<Button> (R.id.btnEnviar)
        txtResultado = findViewById(R.id.resultado)
        botonEnviar.setOnClickListener(this);

    }

    override fun onClick(v: View?) {
        val editTextNombre = findViewById<EditText>(R.id.editTextNombre)
        val editTextApellidos = findViewById<EditText>(R.id.editTextApellidos)

        txtResultado?.setText("Hola ${editTextNombre.text} ${editTextApellidos.text}")
    }
}
